media-capabilities/decodingInfo.any.js

```diff
@@ -180,10 +233,10 @@ promise_test(t => {
       width: 800,
       height: 600,
       bitrate: 3000,
-      framerate: '24000/1001',
+      framerate: 24,
```

Comment:
Was this change intentional?  The test suggests the framerate needs to be 'x/y'

Reply:
Sorry, no, I just fixed that.